src/node_options.cc

```diff
@@ -1216,6 +1216,9 @@ PerProcessOptionsParser::PerProcessOptionsParser(
   AddOption("--run",
             "Run a script specified in package.json",
             &PerProcessOptions::run);
+  AddOption("--run-from",
+            "Run a package.json script from a specific directory",
```

Comment:
If we say path here, it's ambiguous whether we refer to the package.json file or its containing folder (which is good because both are valid).

```suggestion
            "Run a package.json script from a specific path",
```